main.py

```diff
@@ -17,17 +17,16 @@
 # End module imports
 
 # Begin user imports
-#import src.RequestHandler
+from src.server import ChatdServer as Server
 # End user imports
 
 # main
 
```

Comment:
Wrap everything in if __name__ = '__main__' to make the code only run when it is executed as a program